.circleci/config.yml

```diff
@@ -54,8 +66,15 @@ jobs:
       - go/mod-download
       - run: sudo apt-get update
       - run: sudo apt-get install npm
+      - restore_cache:
```

Comment:
Afaik this is already done by `go/mod-download`.

Reply:
it used to be done, but then apparently was removed. - https://circleci.com/orbs/registry/orb/gotest/tools#commands-mod-download

I'll do a quick speed test. we definitely need the cache for mac build because it does a long cargo install

Reply:
Ahh. THanks.